### DISCUSSION TITLE
Shelly 2.5 lifetime

### DISCUSSION THREAD

**Discussion by adamgdynia** [RavenSystem/esp-homekit-devices#1867] [General]
I've got two dead shelly 2.5 devices. Both units failed within a week. What is lifespan of your installation? Mine lasted 2 years. After fresh re-flash I managed to add them to HomeKit, but when I connect 230V, the device is not recognizable, heats up and you may head a hissing sound. The load was insignificant (bedside lamp). Shelly tech support suggested that the unis seem faulty.

I have also Shelly 1 in my "collection" for longer time - I guess those are more durable.

Is there an alternative to Shelly 2.5 (except Sonoff dual R3 which is too bulky and harder to flash)?

**Comment by pawelosin** [comment]
Jeśli chodzi o Selly 2.5 to nie mam z nim doświadczenia, używam wielu Shelly 1 i kilka z nich też mi się uszkodziło, natomiast co do modułu jaki ja używam do lampki nocnej to jest Sonoff basic r3. Jest co prawda jednokanałowy, ale wygląda całkiem ładnie i ma wbudowany przycisk, który da się zaprogramować jako włącznik modułu oraz dodatkowo jako przycisk do sterowania innymi urządzeniami.

**Comment by MagOO33** [comment]
Same thing with one shelly out of seven. The buzzing sound could be a capacitor issue. Some say that it's about replacing the 16V/100uF capacitor.

**Comment by shanghaix** [comment]
The same thing happened to me, after about 2 years, multiple Shelly 2.5 devices started dying (some high pitch whistling sound comes out of the device) in the last couple of weeks. Since they're out of warranty, my only recourse was to buy new ones and flash HAA in them again to replace the broken ones.

**Comment by archiolochos** [comment]
same here. it happend on two of five 2.5 shellys

**Comment by pilot1981** [comment]
in my experience shelly1 is very reliable, but shelly2.5 have a lot a problem, so if you have sufficient space I suggest you to use always shelly1.
On the web you can find many users used capacitor in the circuit with shelly2.5 or other user repaired shelly2.5 changing Installed capacitor...
